lte/gateway/python/magma/pipelined/tests/test_pull_stats.py

```diff
@@ -181,11 +181,7 @@ def test_poll(self):
             self.assertEqual(rule_record_cookie.bytes_tx, 0)
             self.assertEqual(rule_record_cookie.bytes_rx, 0)
             rule_map_cookie_and_mask = self.enforcement_stats_controller.get_stats(1, 1)
```

Comment:
Just make this get_stats(0,0), then you can keep the lovely asserts

Reply:
get_stats(0, 0) is already tested(default arguments test), so I'll just remove this case.